formats/json-refract-schema-tests.json

```diff
@@ -0,0 +1,586 @@
+          "element": "string",
+          "meta": {
+            "id": "Name"
+          }
+        },
+        "valid": false
+      },
+      {
+        "description": "with title metadata",
+        "data": {
+          "element": "string",
+          "meta": {
+            "title": {
+              "element": "string",
+              "content": "Doe"
+            }
+          }
+        },
+        "valid": true
+      },
+      {
+        "description": "with title metadata as non string element",
+        "data": {
+          "element": "string",
+          "meta": {
+            "title": {
+              "element": "number",
+              "content": 1
+            }
+          }
+        },
+        "valid": false
+      },
+      {
+        "description": "with non-element title metadata",
+        "data": {
+          "element": "string",
+          "meta": {
+            "title": "Doe"
+          }
+        },
+        "valid": false
+      },
+      {
+        "description": "with description metadata",
+        "data": {
+          "element": "string",
+          "meta": {
+            "description": {
+              "element": "string",
+              "content": "Doe"
+            }
+          }
+        },
+        "valid": true
+      },
+      {
+        "description": "with description metadata as non string element",
+        "data": {
+          "element": "string",
+          "meta": {
+            "description": {
+              "element": "number",
+              "content": 1
+            }
+          }
+        },
+        "valid": false
+      },
+      {
+        "description": "with non-element description metadata",
+        "data": {
+          "element": "string",
+          "meta": {
+            "description": "Doe"
+          }
+        },
+        "valid": false
+      },
+      {
+        "description": "with unknown meta key",
+        "data": {
+          "element": "string",
+          "meta": {
+            "something": {
+              "element": "string",
+              "content": "Name"
+            }
+          }
+        },
+        "valid": false
+      },
+      {
+        "description": "with array element as classes",
+        "data": {
+          "element": "string",
+          "meta": {
+            "classes": {
+              "element": "array",
+              "content": [
+                {
+                  "element": "string",
+                  "content": "error"
+                }
+              ]
+            }
+          }
+        },
+        "valid": true
+      },
+      {
+        "description": "with array element of non string element as classes",
+        "data": {
+          "element": "string",
+          "meta": {
+            "classes": {
+              "element": "array",
+              "content": [
+                {
+                  "element": "number"
+                }
+              ]
+            }
+          }
+        },
+        "valid": false
+      },
+      {
```

Comment:
That test has classes as an object. I want another test to have just a plain array (our old serialisation form).